concepticondata/conceptlists/Clark-2008-441.tsv

```diff
@@ -0,0 +1,442 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Clark-2008-441-1	1	ant	587	ANT
+Clark-2008-441-2	2	bat	1793	BAT
+Clark-2008-441-3	3	bear	1820	BEAR
+Clark-2008-441-4	4	bee	665	BEE
+Clark-2008-441-5	5	bird	937	BIRD
+Clark-2008-441-6	6	buffalo	934	BUFFALO
+Clark-2008-441-7	7	cat	1208	CAT
+Clark-2008-441-8	8	caterpillar	2247	CATERPILLAR
+Clark-2008-441-9	9	centipede	1974	CENTIPEDE
+Clark-2008-441-10	10	chicken cock	1511	ROOSTER
+Clark-2008-441-11	11	chicken hen	1514	HEN
+Clark-2008-441-12	12	chicken	1318	CHICKEN
+Clark-2008-441-13	13	cow/ox	1153	CATTLE
+Clark-2008-441-14	14	crab	2348	CRAB
+Clark-2008-441-15	15	cricket	2354	CRICKET
+Clark-2008-441-16	16	crow	913	CROW
+Clark-2008-441-17	17	barking deer	3152	MUNTJACS
+Clark-2008-441-18	18	sambar deer		
+Clark-2008-441-19	19	dog	2009	DOG
+Clark-2008-441-20	20	dragon	2038	DRAGON
+Clark-2008-441-21	21	dragon fly	2352	DRAGONFLY
+Clark-2008-441-22	22	duck1	1360	DUCK
```

Comment:
Is it possible to distinguish between duck1 and duck2? Maybe one of them is 3014 | BLACK DUCK ?